src/engine/Map.cxx

```diff
@@ -405,13 +431,21 @@ bool Map::isClickWithinTile(const SDL_Point &screenCoordinates, int isoX, int is
   return false;
 }
 
-void Map::highlightNode(const Point &isoCoordinates)
+void Map::highlightNode(const Point &isoCoordinates, bool redHighlight)
```

Comment:
If we ever want to support more highlight colors, it might be better to just send the color as parameter

Reply:
will fix that in the next PR